Data/future/usa/margins/GC.csv

```diff
@@ -34,3 +34,21 @@ date,initial,maintenance
 20160627,6050,5500
 20160701,6600,6000
 20160811,6600,6000
+20161103,6600,6000
+20161223,6600,6000
+20170113,5940,5400
+20170317,5280,4800
```

Comment:
Missing an entry here, probably is a duplicate of this line since it has the same data at a prior date: https://www.cmegroup.com/notices/clearing/2017/03/Chadv17-085.html#pageNumber=1